### PR TITLE
Add a prompt to allow signing into the frontend

### PR DESCRIPTION
**Pull request by Signez**
At last!

capture d ecran de 2017-01-08 15-07-57

This PR also moves things around to make sure we can use the token generated by the authentication can be used afterward.

@rlustin Please review.

### CODE REVIEW

File: src/frontend/app/components/login-prompt.vue [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      fetch(request)
+        .then((response) => {
+          if (response.ok) {
+            response.json().then((data) => {
+              this.$emit("didLogIn", data.token, data.user_id, data.expired_at)
+
+              // Clean up email/password storage after login
+              this.loginEmail = '';
+              this.loginPassword = '';
+              this.loginError = '';
```
Comment (rlustin) [inline src/frontend/app/components/login-prompt.vue]: Can you fetch translations after login success?

File: src/frontend/app/components/toolbar.vue [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
 </template>
 
 <script>
   export default ({
-    name: 'toolbar'
+    name: 'toolbar',
+
+    methods: {
+      logout() {
+        this.$emit('logout');
```
Comment (rlustin) [inline src/frontend/app/components/toolbar.vue]: Can you call `/api/v1/logout` endpoint to also invalidate the token on the backend side?

File: src/frontend/app/components/login-prompt.vue (hunk 1 above)
Comment (Signez) [inline src/frontend/app/components/login-prompt.vue]: Done in `app.vue`.

File: src/frontend/app/components/toolbar.vue (hunk 2 above)
Comment (Signez) [inline src/frontend/app/components/toolbar.vue]: Done in `app.vue`.

File: src/frontend/app/components/login-prompt.vue (hunk 1 above)
Comment (rlustin) [inline src/frontend/app/components/login-prompt.vue]: 👍

File: src/frontend/app/components/toolbar.vue (hunk 2 above)
Comment (rlustin) [inline src/frontend/app/components/toolbar.vue]: 👍

**Pull request merged**